Commands/Anime/tickle.js

```diff
@@ -12,34 +12,11 @@ module.exports = {
     },
     run: async (client, message, args) => {
         const { url } = await tickle().catch(() => { });
-
         if (!url) return message.channel.send(`Could not connect to nekos.life`);
-
         const embed = new MessageEmbed();
-
-        if (
-            message.mentions.members.size &&
-            message.mentions.members.first().id === client.user.id
-        ) {
-            return message.channel.send(
-                `B~Baka ${message.member}! Stop that~ it tickles!`
-            );
-        } else if (
-            message.mentions.members.size &&
-            message.mentions.members.first().id === message.author.id
-        ) {
-            return message.channel.send(`Wai~ Seriously!?`);
-        } else if (message.mentions.members.size) {
-            return message.channel.send(
-                embed
-                    .setColor(config.embedcolor)
-                    .setDescription(`${message.member} started tickling ${message.mentions.members.first()}!`)
-                    .setImage(url)
-            );
-        } else {
-            return message.channel.send(
-                embed.setColor(config.embedcolor).setImage(url)
-            );
-        }
+        if (message.mentions.members.size && message.mentions.members.first().id === client.user.id) return message.channel.send(`B~Baka ${message.member}! Stop that~ it tickles!`); 
+        if (message.mentions.members.size && message.mentions.members.first().id === message.author.id) return message.channel.send(`Wai~ Seriously!?`);

```

Comment:
You should use 
```js
message.channel.send({content: ""})
```

Reply:
If it's string only, you can use message.channel.send("string")